config/webpack.config.js

```diff
@@ -245,41 +251,94 @@ module.exports = env => {
         fileName: 'file-manifest.json',
       }),
     );
-  } else {
+  }
+
+  // Optionally generate landing pages in the absence of a content build.
+  if (buildOptions.scaffold) {
     const landingPagePath = rootUrl =>
       path.join(outputPath, '../', rootUrl, 'index.html');
 
+    const inlineScripts = [
+      'incompatible-browser.js',
+      'record-event.js',
+      'static-page-widgets.js',
+    ].reduce(
+      (scripts, filename) => ({
+        ...scripts,
+        [filename]: fs.readFileSync(path.join('src/site/assets/js', filename)),
+      }),
+      {},
+    );
+
+    // Modifies the style tags output from HTML Webpack Plugin.
+    const modifyStyleTags = pluginStyleTags =>
+      pluginStyleTags
+        .reduce(
+          (tags, tag) =>
+            // Puts style.css before the app-specific stylesheet.
+            tag.attributes.href.match(/style/)
+              ? [tag, ...tags]
+              : [...tags, tag],
+          [],
+        )
+        .join('');
```

Comment:
It isn't clear to me why we're doing this, and additionally it seems less like we're modifying the tags and more like we're reordering them.

Reply:
The plugin outputs the tags in an array in a certain order, but we're reordering them to keep the order consistent with actual content pages, in case the order might matter. From my testing, it didn't seem to matter with the default order, but there's not really a performance hit, and the consistency with real content wouldn't hurt.

I used `modify` since it's changing what the plugin is normally outputting, but I'm open to naming suggestions.

Reply:
The default style tags output seems to put the app style (e.g., `hca.css`) before `style.css`, but naturally, we want `style.css` to be first in case apps are overriding some default.

The default script tags output is `vendor`, `polyfills`, `style`, then app script (e.g. `hca.entry.js`). `polyfillls` should go before `vendor`, but I couldn't get the plugin to reorder things properly, even with `chunksSortMode: 'manual'`.

And the reason I'm leaving the tags output up to the plugin is that it automatically handles the hashed file names for optimized builds.

Reply:
TLDR: `chunksSortMode: 'manual'` doesn't work (and neither does `'none'`), so this is a hack to ensure the proper order.